pkg/execution/state/redis_state/queue.go

```diff
@@ -1180,6 +1184,7 @@ func (q *queue) EnqueueItem(ctx context.Context, i QueueItem, at time.Time) (Que
 		parts[0].ID,
 		parts[1].ID,
 		parts[2].ID,
+		i.Data.Identifier.AccountID.String(),
```

Comment:
Are we sure there are no more queue items without an Identifier? We might need to check this!